site/src/sphinx/client-circuit-breaker.rst

```diff
@@ -111,40 +110,91 @@ is raised or the status is ``5xx``, succeeds when the status is ``2xx`` and igno
-                    if (status.codeClass() == HttpStatusClass.SUCCESS) {
-                        return true;
-                    }
+                // A success if the response is 2xx.
+                if (status.codeClass() == HttpStatusClass.SUCCESS) {
+                    return CompletableFuture.completedFuture(true);
                 }
+            }
 
-                // Neither a success nor a failure. Do not take this response into account.
-                return null;
-            });
+            // Neither a success nor a failure. Do not take this response into account.
+            return CompletableFuture.completedFuture(null);
         }
     };
 
 If you want to treat a :api:`Response` as a success, return ``true``. Return ``false`` to treat as a failure.
 Note that :api:`CircuitBreakerStrategy` can return ``null`` as well. It won't be counted as a success nor
 a failure.
 
+If you need to determine whether the request was successful by looking into the response content,
+you should implement :api:`CircuitBreakerStrategyWithContent` and specify it when you create an
+:api:`HttpClient` using :api:`CircuitBreakerHttpClientBuilder`:
+
+.. code-block:: java
+
+    import com.linecorp.armeria.client.circuitbreaker.CircuitBreakerStrategyWithContent;
+
+    final CircuitBreakerStrategyWithContent<HttpResponse> myStrategy =
+            new CircuitBreakerStrategyWithContent<HttpResponse>() {
+
+                @Override
+                public CompletionStage<Boolean> shouldReportAsSuccess(ClientRequestContext ctx,
+                                                                      HttpResponse response) {
+                    return response.aggregate().handle((res, cause) -> {
+                        if (cause != null) {
+                            if (cause instanceof UnprocessedRequestException) {
+                                // Neither a success nor a failure because the request has not been handled
+                                // by the server.
+                                return null;
+                            }
+                            // A failure if an Exception is raised.
+                            return false;
+                        }
+
+                        final String content = res.content().toStringUtf8();
+                        if ("Success".equals(content)) {
+                            return true;
+                        } else if ("Failure".equals(content)) {
+                            return false;
+                        }
+
+                        // Neither a success nor a failure. Do not take this response into account.
+                        return null;
+                    });
+                }
+            };
+
+    final HttpClient client = new HttpClientBuilder(...)
+            .decorator(new CircuitBreakerHttpClientBuilder(myStrategy).newDecorator()) // Specify the strategy
+            .build();
+
+    client.execute(...).aggregate().join();
```

Comment:
`final AggregatedHttpResponse res = client.execute(...)...;`

Reply:
Fixed Thank you~!